src/query/api/v1/handler/placement/common.go

```diff
@@ -479,3 +481,63 @@ func isStateless(serviceName string) bool {
 	}
 	return false
 }
+
+func deleteAggregatorShardSetIDRelatedKeys(
+	svc handleroptions.ServiceNameAndDefaults,
+	svcOpts handleroptions.ServiceOptions,
+	clusterClient clusterclient.Client,
+	shardSetIDs []uint32,
+) error {
+	if svc.ServiceName != handleroptions.M3AggregatorServiceName {
+		return fmt.Errorf("error deleting aggregator instance keys, bad service: %s",
+			svc.ServiceName)
+	}
+
+	kvOpts := kv.NewOverrideOptions().
+		SetEnvironment(svcOpts.ServiceEnvironment).
+		SetZone(svcOpts.ServiceZone)
+
+	flushTimesMgrOpts := aggregator.NewFlushTimesManagerOptions()
+	electionMgrOpts := aggregator.NewElectionManagerOptions()
```

Comment:
meganit: do this after getting the store?

Reply:
Also, are these guaranteed to be default?

Reply:
Sure thing.